cmd/influx/main.go

```diff
@@ -249,6 +258,22 @@ func (c *CommandLine) SetFormat(cmd string) {
 	}
 }
 
+func (c *CommandLine) dump() {
```

Comment:
I'm ok with the `dump()` naming but the `Dump` flag doesn't indicate in its name that it's a boolean. I would suggest either naming the flag `DumpEnabled` or `ShouldDump`. Also, naming two things on a struct the same name is generally confusing.